### PR TITLE
https://issues.redhat.com/browse/ACM-14719 InfraEnv known issue

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.11, 2.10 and 2.9

### CODE REVIEW

**Review by CrystalChun — APPROVED** [review]
/lgtm

File: clusters/release_notes/mce_known_issues.adoc [hunk 1]
```diff
@@ -360,6 +360,12 @@ If this problem occurs, it is typically on the following versions of {ocp-short}
 
 To avoid this error, upgrade your {ocp-short} to version 4.8.18 or later, or 4.9.7 or later.
 
+[#boot-discovey-auto-add-host]
+=== Cannot use host inventory for booting with the discovery image and adding hosts automatically
```
Comment (swopebe) [inline clusters/release_notes/mce_known_issues.adoc]:
```suggestion
=== Cannot use host inventory to boot with the discovery image and add hosts automatically
```

File: clusters/release_notes/mce_known_issues.adoc [hunk 2]
```diff
@@ -360,6 +360,12 @@ If this problem occurs, it is typically on the following versions of {ocp-short}
...
+//2.12:ACM-14719
+
+You cannot use a host inventory, or `InfraEnv` custom resource, for both booting with the discovery image and adding hosts automatically. If you used your old `InfraEnv` for `BareMetalHost` and want to boot the image yourself, you can work around the issue by creating a new `InfraEnv`.
```
Comment (swopebe) [inline clusters/release_notes/mce_known_issues.adoc]: Let's not use old if we can avoid it (?); I have mentioned in meetings in the past and will mention again, better to use _previous_. I have always been told calling customers components and what not "old" can insult users who are not on the latest release or version on that component. It carries negative connotation. I know this isn't a legacy product where old means "you need to upgrade" but I think it is just good practice.

Comment (swopebe) [inline clusters/release_notes/mce_known_issues.adoc]:
```suggestion
You cannot use a host inventory, or `InfraEnv` custom resource, to both boot with the discovery image and add hosts automatically. If you used your previous `InfraEnv` resource for the `BareMetalHost` resource, and you want to boot the image yourself, you can work around the issue by creating a new `InfraEnv` resource.
```

Comment (swopebe) [reply clusters/release_notes/mce_known_issues.adoc]: Are there any steps for this or are we assuming they know how from previous procedures they used?

Comment (swopebe) [reply clusters/release_notes/mce_known_issues.adoc]: comma here?

_image, and add_

**Review by swopebe — APPROVED** [review]
I won't hold it up contingent on the few improvements.

File: clusters/release_notes/mce_known_issues.adoc (hunk 2 above)
Comment (oafischer) [reply clusters/release_notes/mce_known_issues.adoc]: Steps for this are in the main AI docs. At this point users have already created the CR, so I think it's fair to assume they know to create another one. If necessary we can link to the main doc though.

Comment (oafischer) [reply clusters/release_notes/mce_known_issues.adoc]: Changed it to previous

**Comment by oafischer** [comment]
Made all the requested changes, merging

**Pull request merged**